README.md

```diff
@@ -115,25 +116,14 @@ Host db-server-1.ap-southeast-1
 
 `ec2ssh remove` command removes the mark lines.
 
-# How to upgrade from 1.x to 2.x
-If you have used ec2ssh-1.x, it seems that you may not have '~/.ec2ssh'.
-So you need execute `ec2ssh init` once to create `~/.ec2ssh`, and edit it as you like.
+# How to upgrade from 3.x
+Dotfile (`.ec2ssh`) format has been changed from 3.x.  
 
-```
-$ ec2ssh init
-$ vi ~/.ec2ssh
-```
-
-# How to upgrade from 2.x to 3.x
-Dotfile (`.ec2ssh`) format has been changed from YAML to Ruby DSL.
-
-Don't panic and run `ec2ssh migrate` if you have ec2ssh-2.x styled dotfile.
-
-```
-$ ec2ssh migrate
-```
+* A instance tag access I/F has been changed from `tags['Name']` to `tag('Name')`
+* `Aws::EC2::Instance` methods have been changed to AWS SDK v2
+* The `aws_keys` structure have benn changed
+  * `aws_keys[profile_name][region] # => Aws::Credentials`
```

Comment:
This is a destructive change, so I need it to cause an error message if user didn't call `aws_keys` without region.

Reply:
Is this mean that print warning if `aws_keys = { "profile1" => Aws::Credentials.new("key", "sec")` is called?

Reply:
I changed at ab0dcfab6c8b5eed57f6d075013a8ba1d292af57

Reply:
> Is this mean that print warning if aws_keys({ "profile1" => { access_key_id: "key",secret_access_key: "sec" }) is called?

Yes!